src/release-verification.js

```diff
@@ -64,22 +64,31 @@ module.exports = robot => {
         'release-pr-prerelease': String(prerelease),
       },
     };
-    const curlCommand = `curl \
-    -H "Authorization: Bearer ${process.env.BUILDKITE_TOKEN}" \
-    -X POST "https://api.buildkite.com/v2/organizations/uberopensource/pipelines/fusion-release-verification/builds" \
-      -d '${JSON.stringify(curlObject)}'`;
-    exec(curlCommand, (error, stdout) => {
-      if (error !== null) {
-        // eslint-disable-next-line no-console
-        console.warn('exec error: ' + error);
-      }
-      const output = JSON.parse(stdout);
-      github.issues.createComment(
-        context.issue({
-          body: `Triggered Fusion.js build verification: ${output.web_url}`,
-        })
+
+    let output;
+
+    try {
+      const res = await fetch(
+        'https://api.buildkite.com/v2/organizations/uberopensource/pipelines/fusion-release-verification/builds',
+        {
+          method: 'POST',
+          body: JSON.stringify(payload),
+          headers: {
+            Authorization: `Bearer ${process.env.BUILDKITE_TOKEN}`,
+          },
+        },
       );
-    });
+      output = await res.json();
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.warn(err);
+    }
+
+    github.issues.createComment(
+      context.issue({
+        body: `Triggered Fusion.js build verification: ${output.web_url}`,
+      }),
```

Comment:
Should this set the status message instead of commenting? @KevinGrandon

Reply:
Yeah, status URL is probably better here, happy to do that in a follow-up.